### PR TITLE
New Cox adapter

### PR DESCRIPTION
**Pull request by reynold-cox**
## Type of change
New bidder adapter

## Description of change
test parameters for validating bids:
```
{
  bidder: 'cox',
  params: {
    size: '300x250',
    id: 2000005991707,
    siteId: 2000100948180,
  }
}
```
- contact email of the adapter’s maintainer: reynold@coxmt.com
- [x ] official adapter submission

### CODE REVIEW

File: src/adapters/cox.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+        //    // If it's still too high, just take 80% to 90% of the bid
+        //    if (floor >= bid) floor = bid * ((Math.random() * 10) + 80) / 100;
+        //  }
+        //  return Math.round(floor * 100) / 100;
+        // },
+
+        // setAuctionPrice: function setAuctionPrice(ad, bid) {
+        //  return ad ? ad.replace('${AUCTION_PRICE}', bid) : ad;
+        // },
```
Comment (reynold-cox) [inline src/adapters/cox.js]: We have a partner that utilizes the commented-out code.  We'd rather keep it in there for completeness' sake.  But if y'all would prefer it be removed, we'll be happy to do so.  Please let me know.  Thanks!

**Comment by dbemiller** [comment]
@reynold-cox What exactly does it mean if someone configures your bidder with a `size` param?

Not that I want to go policing bidders' params or anything... but `size` already comes in on the bid, and nobody reviewing this could think of a reason why a _bidder_ would have a size... so I wanted to make sure it wasn't an oversight & duplicate info.

Or if there's motivation for the duplicate info, then it'd be very helpful to know why, so we can factor the use case into future prebid designs.

**Comment by protonate** [comment]
I'm not able to get bids back, are the test params still operational?

**Comment by reynold-cox** [comment]
Hi David
Our ad spaces only support a single size each, and our ad servers expect the size to be present in the request.  Since the ad unit config can have multiple sizes defined, we can't always deduce the size for a given Cox ad space...so that's why we have the "size" key in the bidder params.  Thanks!

**Comment by reynold-cox** [comment]
@protonate:
The test params should be ok.  I just used the following and I'm getting a non-zero bid back:

    {
      bidder: 'cox',
        params: {
          size: '300x250',
          id: 2000005991707,
          siteId: 2000100948180,
      }
    }

Are you not getting a response, or are you getting a zero bid?

**Comment by protonate** [comment]
Ah, I see, thanks, it works if running on `localhost`, confirmed bids back, reviewed.

**Pull request merged**